portal-api/test-helpers/unit-test-mocks/mock-user.js

```diff
@@ -15,44 +15,40 @@ const TEST_USER = {
     name: 'HSBC',
     emails: ['maker1@ukexportfinance.gov.uk', 'maker2@ukexportfinance.gov.uk'],
   },
-  signInToken: { salt: Buffer.from('03', 'hex'), hash: Buffer.from('04', 'hex') },
 };
 
-const TEST_DATABASE_USER = {
-  _id: '075bcd157dcb851180e02a7c',
-  'user-status': STATUS.ACTIVE,
-  timezone: 'Europe/London',
-  username: 'HSBC-maker-1',
-  firstname: 'Mister',
-  surname: 'One',
-  email: 'one@email.com',
-  roles: [MAKER],
-  bank: {
-    id: '961',
-    name: 'HSBC',
-    emails: ['maker1@ukexportfinance.gov.uk', 'maker2@ukexportfinance.gov.uk'],
-  },
-  salt: '01',
-  hash: '02',
-  signInToken: { saltHex: '03', hashHex: '04', expiry: 1700501222290 },
-};
+const TEST_USER = produce(BASE_TEST_USER, (draft) => {
```

Comment:
I've cleared this file up with test users for specific use cases. Should make our testing more reliable and easy to maintain. Some existing code debt that I have not had time to solve in this ticket is this `TEST_USER` has a very odd payload, and is used thoughout tests. We should replace this eventually with more specific test users.